### PR TITLE
Add unit test for cli module

### PR DESCRIPTION
**Pull request by cescgina**
Add unit tests for cli module. CI checks will fail due to low coverage of the
commands moudules.

### CODE REVIEW

**Review by jsanemet — APPROVED** [review]
Looks good, great job.

**Review by adrianfusco — DISMISSED** [review]

**Comment by jsanemet** [comment]
Off-topic, maybe it is a good idea to remove the need for a 90% coverage until testing has matured a bit more. Right now we are not going to achieve that goal and all pull requests will have their checks failing.

We can still enforce that rule by taking a look during review.

**Review by sdatko — APPROVED** [review]
:+1: from me, good job!

```
Name                        Stmts   Miss  Cover
-----------------------------------------------
(...)
znoyder/cli.py                108     11    90%
(...)
```

File: tests/test_cli.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+import os
+from unittest import TestCase
+
+from znoyder.cli import process_arguments
+
+
+class TestCli(TestCase):
+    """
+        Test cli for znoyder.
+    """
```
[the remark this answers is not in the document]
Comment (cescgina) [reply tests/test_cli.py:27]: You're, it should be a single line, I'll upload with a quick fix

**Comment by sdatko** [comment]
> Off-topic, maybe it is a good idea to remove the need for a 90% coverage until testing has matured a bit more. Right now we are not going to achieve that goal and all pull requests will have their checks failing.

We are getting closer and closer to that, I think we may reach that quite soon, so I would stick with this goal still.

**Review by sdatko — APPROVED** [review]
Thanks for amazing work there!

**Pull request merged**